Testscripts/Linux/customKernelInstall.sh

```diff
@@ -17,7 +17,7 @@
 
 supported_kernels=(ppa proposed proposed-azure proposed-edge latest
                     linuxnext netnext upstream-stable)
-
+tarDestination="./linux-source"
```

Comment:
Where does it come from?

Reply:
I included it to keep a common destination after tar file is extracted. Why?